includes/admin/class-wcmp-export-consignments.php

```diff
@@ -337,8 +335,21 @@ private function getInsurance(): int
      */
     private function getInsuranceAmount($isInsuranceActive)
     {
+        // Checks if all parcels must be insured
         if ($isInsuranceActive) {
-            return $this->getSetting("{$this->carrier}_" . WCMP_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED_AMOUNT);
+            // get min price for insurance
+            $insuranceFromPrice = (int) $this->getSetting("{$this->carrier}_" .
+                WCMP_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED_FROM_PRICE
+            );
+
+            // get the order's total price
+            $orderPrice = (int) $this->order->get_total();
```

Comment:
Is dit niet een float? Heeft het centen?

Reply:
$insuranceFromPrice = (int) 2.10
$orderPrice = 2
if ($insuranceFromPrice <= $orderPrice) // true, terwijl je false verwacht

Reply:
Klopt. Heb het aangepast!